wiki/Help_Centre/en.md

```diff
@@ -628,23 +646,59 @@ You will need to consult your tablet's manufacturer to find out how to set your
 
 All osu! branded tablets support raw input by default, and may have their sensitivity easily adjusted on the fly by using the raw input option in game and using the mouse sensitivity slider.
 
-#### My song list keeps scrolling!
+#### I'm experiencing lag spikes or stuttering during play!
 
-**This is usually due to a faulty input device connected to your computer. Try unplugging any controllers or joysticks.**
+**There's many reasons why the game may perform inconsistently, from programs on your computer, to out-of-date drivers, or simply having some settings too high.**
 
-Applications which emulate or change keybindings such as *Xpadder* or *X-Mouse Button Control* may also cause these issues. Try disabling them if you use them for other games.
+##### Do you have lots of programs open at the same time while playing osu!?
 
-This can also happen due to issues with the numpad keys, as they can be used to scroll the song selection list. Pressing the `NumLock` key to disable the numpad and then pressing either `9` or `3` on the numpad will fix the issue.
+**Some background processes can fight with osu! for resources and make the game very unpleasant to play.**
 
-You may also wish to check for any jammed or damaged buttons on your peripherals.
+Try closing some programs that you do not need open while playing the game and/or all of them.
 
-#### I downloaded a Beatmap Pack but osu! is always trying to "repair" the file!
+##### Is your computer secure?
 
-**You will need to extract the pack into your Songs directory.**
+**Viruses, spyware, malware or other unwanted software can cause serious performance problems for even high-end computers.**
 
-Most beatmap packs come in a `.rar` file format, which means they are a compressed archive of beatmap files. You will need to extract them using your utility of choice (we recommend [7-Zip](https://7-zip.org)) to your Songs directory first.
+Keep your computer clean and safe and you might notice an improvement. Try running a virus scan on your computer to find and get eleminate such issues.
```

Comment:
```suggestion
Keep your computer clean and safe and you might notice an improvement. Try running a virus scan on your computer to find and get eliminate such issues.
```

Reply:
typo oop

Reply:
I'm dumb, will double check for spelling errors, thanks.